ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactionsTest.java

```diff
@@ -409,4 +412,58 @@ private Transaction createTransaction(final int transactionNumber) {
         .nonce(transactionNumber)
         .createTransaction(KEYS1);
   }
+
+  @Test
+  public void shouldEvictMultipleOldTransactions() {
+    final long transactionEvictionIntervalMs = 1L;
+    final TestClock clock = new TestClock();
```

Comment:
nit: Make clock a field so it can be reused.

I'd also stick with the default 1 minute timeout for transactions - we can step the clock forward as much as we like so don't need to have a custom value here and can then reuse the `PendingTransactions` instance in the field.